python/res/enkf/jobs/enkf_scaling_job.py

```diff
@@ -0,0 +1,220 @@
+# -*- coding: utf-8 -*-
+
+import sys
+import configsuite
+
+from copy import deepcopy
+from collections import namedtuple
+
+from res.enkf.jobs import enkf_scaling_job_config
+from res.enkf.jobs.enkf_scaling_pca import get_scaling_factor
+from res.enkf import LocalObsdata, ActiveList
+from res.enkf.jobs.enkf_scaling_get_data import get_measured_data, get_scaled_matrix
+
+
+def main_entry_point(ert, user_config_dict):
```

Comment:
I would restructure this where main_entry_point is renamed to ..._scale_job and the verification of the config is moved, if needed, to a different function

Reply:
:+1: